cmd/productinfo/main.go

```diff
@@ -66,6 +69,8 @@ func defineFlags() {
 	flag.StringSlice(providerFlag, []string{Ec2, Gce, Azure}, "Providers that will be used with the productinfo application.")
 	flag.String(azureSubscriptionId, "", "Azure subscription ID to use with the APIs")
 	flag.Bool(helpFlag, false, "print usage")
+	flag.Bool(metricsEnabledFlag, false, "if true get prometheus metrics, false by default")
```

Comment:
please change description to `internal metrics are exposed if enabled`

Reply:
fixed